dts/bindings/base/base.yaml

```diff
@@ -109,3 +109,21 @@ properties:
       required: false
       type: string-array
       description: Provided names of mailbox / IPM channel specifiers
+
+    zephyr,memory-region:
+      type: string
+      required: false
+      description: |
+        Signify that this node should result in a dedicated linker script
+        memory region in the final executable. The region address and size
```

Comment:
This is actually only true for architectures supporting this. ARM64 is not one of those. Can you expand this with:

`Signify that this node should result in a dedicated linker script memory region in the final executable when supported by the linker script`.

Reply:
Updated, but replace linker with build.